src/lib/components/html-report-page.jsx

```diff
@@ -20,37 +20,55 @@ export type FlowUncoveredLocsProps = {
   uncovered_locs: Array<FlowUncoveredLoc>
 };
 
-/* eslint-disable react/no-unused-prop-types */
-export type FlowCoverageReportProps = {
-  reportType: 'summary' | 'sourcefile',
+export type FlowReportMetaProps = {
+  assets: {
+    css?: Array<string>,
+    js?: Array<string>
+  },
   coverageGeneratedAt: string,
+  htmlTemplateOptions: {
+    autoHeightSource?: boolean,
+  },
+};
+
+export type FlowCoverageSummaryReportProps = {
   assets: {
     css?: Array<string>,
     js?: Array<string>
   },
+  coverageGeneratedAt: string,
+  htmlTemplateOptions: {
+    autoHeightSource?: boolean,
+  },
```

Comment:
@ryan953 Looks great! Thanks!

Nit: How about renaming the base type from `*Meta*` into `*Base*`?